### PR TITLE
Fixing documentation inconsistency for PTO-Sim blocks

### PR DESCRIPTION
**Pull request by jleonqu**
This PR is related to issue #1314 
The comments for the PTO-Sim class have been revised and updated

### CODE REVIEW

File: source/objects/ptoSimClass.m [hunk 1]
```diff
@@ -94,16 +94,16 @@
...
-            'Amin',                             'NOT DEFINED',...           % Minimum opening area of the valve
-            'pMax',                             'NOT DEFINED',...           % Pressure at maximum opening
-            'pMin',                             'NOT DEFINED',...           % Cracking pressure
-            'rho',                              'NOT DEFINED',...           % Fluid density
-            'k1',                               'NOT DEFINED',...           % Valve coefficiente
-            'k2',                               'NOT DEFINED')              % Valve coefficient, it's a function of the other valve variables
-        simpleDirectDrivePTO (1,1) struct      = struct(...                 % simple direct drive linear PTO Block properties
+        rectifyingCheckValve (1,1) struct       = struct(...                % Rectifying Check Valve Block properties
+            'Cd',                               'NOT DEFINED',...           % [1] Discharge coefficient
+            'Amax',                             'NOT DEFINED',...           % [m^2] Maximum opening area of the valve
```
Comment (kmruehl) [inline source/objects/ptoSimClass.m:99]: @jleonqu this looks good, thank you. Can you clarify the `[1]` in front of `Discharge Coefficient` and `Valve Coefficient`?

**Review by kmruehl — COMMENTED** [review]
See above quesiton

File: source/objects/ptoSimClass.m (hunk 1 above)
Comment (jleonqu) [reply source/objects/ptoSimClass.m:99]: @kmruehl These are nondimensional values. I added a comment to all the nondimensional values for clarification.

**Comment by MShabara** [comment]
@jleonqu Thank you for the PR, I reviewed it and its good to be merged.

**Pull request merged**